File: src/main/java/baboon/industry/block/energy/cables/entity/TileEntityCable.java
package baboon.industry.block.energy.cables.entity;

import baboon.industry.block.energy.entity.TileEntityEnergyConductorDamageable;
import sunsetsatellite.catalyst.core.util.Connection;
import sunsetsatellite.catalyst.core.util.Direction;

public class TileEntityCable extends TileEntityEnergyConductorDamageable {
    public TileEntityCable(){

    }

    public TileEntityCable(int capacity, int energy, int tranfer, int dangerLevel) {
        setCapacity(capacity);
        setEnergy(energy);
        setTransfer(tranfer);

        for (Direction dir : Direction.values())
            setConnection(dir, Connection.BOTH);
    }
}

File: src/main/kotlin/turniplabs/industry/blocks/entities/TileEntityCable.kt
package turniplabs.industry.blocks.entities

import sunsetsatellite.sunsetutils.util.Connection
import sunsetsatellite.sunsetutils.util.Direction

class TileEntityCable(capacity: Int, energy: Int, transfer: Int, var cableDangerLevel: Int) :
    TileEntityEnergyConductorDamageable() {

    init {
        setCapacity(capacity)
        setEnergy(energy)
        setTransfer(transfer)

        for (dir in Direction.values())
            setConnection(dir, Connection.BOTH)
    }
}